cmd/account/console.go

```diff
@@ -104,14 +107,19 @@ func (o *consoleOptions) run() error {
 
 	ctx := context.TODO()
 	var accountID string
+	var isBYOC bool
+	var acctSuffix string
 	if o.accountName != "" {
 		account, err := k8s.GetAWSAccount(ctx, o.kubeCli, o.accountNamespace, o.accountName)
 		if err != nil {
 			return err
 		}
 		accountID = account.Spec.AwsAccountID
+		isBYOC = account.Spec.BYOC
+		acctSuffix = account.Labels["iamUserId"]
```

Comment:
It would be good to make sure this isn't blank, but its a nice to have, not required.

Reply:
Because of the scope of the `account` var here I am just setting the variable here and I'm checking later: https://github.com/openshift/osd-utils-cli/pull/68/files/e9bb1f87dfdd93402bad49af075503753dd604b8#diff-0749f070a0fba513d2ec280163a6951aR157-R161